src/runtime/hexagon_remote/Makefile

```diff
@@ -78,7 +78,8 @@ hosts: \
 remotes: \
 	bin/v60/libhalide_hexagon_remote_skel.so \
 	bin/v60/signed_by_debug/libhalide_hexagon_remote_skel.so \
-	bin/v60/hexagon_sim_remote
+	bin/v60/hexagon_sim_remote \
+	bin/v60/sim_qurt/libsim_qurt.a
```

Comment:
I think this should probably not be in a subfolder, just in `bin/v60` directly

Reply:
Moved it to bin/v60